qa/acceptance/Vagrantfile

```diff
@@ -27,8 +27,8 @@ end
 
 def common_config(machine, type)
   machine.vm.provider "virtualbox" do |v|
-    v.memory = 4096
```

Comment:
good catch, this is a leftover (_i actually never removed_) from where I pull this config files (aka also running elasticsearch so need more mem).